project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -116,7 +119,7 @@ A project may request an allocation from the travel fund as follows:
 * At the completion of the event/allocation period or when re-imbursement
   is to be paid out the project will PR in the approvals into the
   appropriate year's file. This PR will be approved and landed by CPC
-  members. Due to the pre-approval of the request from the project
+  members. Due to the pre*approval of the request from the project
```

Comment:
```suggestion
  members. Due to the pre-approval of the request from the project
```

Reply:
Thank you I have already fixed those in the last commit

Reply:
these are still not fixed.

Reply:
just pushed it